docs/src/main/sphinx/sql/select.md

```diff
@@ -379,6 +379,30 @@ Complex grouping operations are often equivalent to a `UNION ALL` of simple
 does not apply, however, when the source of data for the aggregation
 is non-deterministic.
 
+### AUTO
+
+`AUTO` syntax implicitly groups on every column not included in an aggregation 
+function.
+
+This query sums the account balance (`acctbal`) for each market segment 
+(`mktsegment`), with `AUTO` automatically determining the grouping 
+keys:
```

Comment:
This paragraph describes the query, but does not explain the conclusion that AUTO derives `mktsegment` as the grouping key since it's not part of any aggregate function. Maybe rephrase to:

> This example query calculates the total account balance per market segment. The `AUTO` clause derives `mktsegment` as the grouping key, since it is not used in any aggregate function (i.e., `sum`).